sdk/src/camera_factory.cpp

```diff
@@ -0,0 +1,21 @@
+#include "camera_factory.h"
+#include "camera_1gen.h"
+
+#include <aditof/device_definitions.h>
+
+namespace aditof {
+
+Camera *CameraFactory::buildCamera(DeviceInterface *device) {
+    using namespace aditof;
+
+    DeviceDetails devDetails;
+    device->getDetails(devDetails);
+    switch (devDetails.sensorType) {
```

Comment:
is this for future usage? as there is only 1 case, why switch?

Reply:
Yes, it is for future usage to support at least another type of camera.